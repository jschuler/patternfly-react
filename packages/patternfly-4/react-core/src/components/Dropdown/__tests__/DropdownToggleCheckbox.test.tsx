// Auto-generated do not edit

import * as React from 'react';
import { mount } from 'enzyme';
import { DropdownToggleCheckbox } from '../DropdownToggleCheckbox';

describe('DropdownToggleCheckbox test', () => {
  it('DropdownToggleCheckbox should match snapshot', () => {
    const view = mount(
      <DropdownToggleCheckbox
        className={"''"}
				isValid={true}
				isDisabled={false}
				isChecked={undefined}
				checked={undefined}
				onChange={() => undefined as any}
				children={undefined}
				id={"'defaultString'"}
				aria-label={"'defaultString'"}
      />);
    expect(view).toMatchSnapshot();
  });
});
