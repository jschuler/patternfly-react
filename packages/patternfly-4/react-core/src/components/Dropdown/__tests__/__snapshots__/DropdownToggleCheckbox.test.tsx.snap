// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownToggleCheckbox test DropdownToggleCheckbox should match snapshot 1`] = `
<DropdownToggleCheckbox
  aria-label="'defaultString'"
  className="''"
  id="'defaultString'"
  isDisabled={false}
  isValid={true}
  onChange={[Function]}
>
  <label
    className="pf-c-dropdown__toggle-check ''"
    htmlFor="'defaultString'"
  >
    <input
      aria-invalid={false}
      aria-label="'defaultString'"
      disabled={false}
      id="'defaultString'"
      type="checkbox"
    />
  </label>
</DropdownToggleCheckbox>
`;
